bot/extensions/challenges/events.py

```diff
@@ -42,6 +45,10 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
         if payload.emoji != self.challenge_reaction:
             return
 
+        member = self.bot.guild.get_member(payload.user_id)
+        if member is None or member.bot:
+            return
+
```

Comment:
We just need to check if the user that reacted is our bot, no other bot will react to the message